scripts/im_plot.sl

```diff
@@ -31,51 +30,61 @@ echo ___im plot___
 
 python $gmsim/workflow/scripts/cybershake/add_to_mgmt_queue.py $MGMT_DB_LOC/mgmt_db_queue $SRF_NAME IM_plot running $SLURM_JOB_ID
 
-res=`python $gmsim/visualization/im/spatialise_im.py $CSV_PATH $STATION_FILE_PATH --out_dir $OUTPUT_XYZ_DIR`
-exit_val=$?
-
-
-if [[ $exit_val == 0 ]]; then
-    echo "finished making csvs for im_plot"
-
-    echo "start plotting stations"
-    
-    failed=0
-    fail_msgs=()
-    success_msgs=()
-    for f in $OUTPUT_XYZ_DIR/*.xyz; do
-        cd $OUTPUT_XYZ_DIR
-        out_dir=$(basename $f | cut -d'.' -f1)
-        mkdir -p $out_dir
-        cd $out_dir
-        if [[ "$f" == *"real"* ]]; then
-          res2=$(python $gmsim/visualization/sources/plot_items.py -t $SRF_NAME --xyz $f -c "$SRF_PATH" --xyz-cpt hot --xyz-cpt-invert --xyz-transparency 30 --xyz-size 0.1 --xyz-cpt-labels `cat $OUTPUT_XYZ_DIR/im_order.txt` -n 2)
-        else
-          res2=$(python $gmsim/visualization/sources/plot_items.py -t $SRF_NAME --xyz $f -c "$SRF_PATH" --xyz-model-params $MODEL_PARAMS --xyz-grid --xyz-grid-search 12m --xyz-landmask --xyz-cpt hot --xyz-cpt-invert --xyz-grid-contours --xyz-transparency 30 --xyz-size 1k --xyz-cpt-labels `cat $OUTPUT_XYZ_DIR/im_order.txt` -n 2)
-        fi
-        exit_val2=$?
-        if [[ $exit_val2 != 0 ]]; then
-            failed=1
-            failed_msgs+=( "failed $res2" )
-        else
-            success_msgs+=( "successfully $res2" )
-        fi 
-    done
+COMPS = `cat $CSV_PATH |cut  -d , -f 2 |sort |uniq` # find what components are used
```

Comment:
Originally, as per the slack clarification it was just adding RotD (if available) as it is going to be our component of interest going forward.

If we want to do multiple components, it might be worth specifying it in the config file - but beyond the scope of this PR.

In general, we wouldn't need the map plots for 000, 090, ver so seems overkill to generate these all the time if they are in the IM_csv

Reply:
Is there a possibility when you (or who else?) do IM calc with 000 090, ver, geom and rotd50 all, then only wish to plot geom and rotd50?

Reply:
Conversation is slightly out of date but currently the default components calculated are:
![image](https://user-images.githubusercontent.com/25143301/135920077-3a377109-1afb-45c3-9c81-a63329184d00.png)

Which is a bit more than what most runs need.

Reply:
Roger that